kolibri/plugins/learn/assets/src/views/assessment-wrapper/index.vue

```diff
@@ -389,7 +389,7 @@ oriented data synchronization.
         if (this.firstAttemptAtQuestion) {
           this.updateAttemptLogMasteryLog({
             correct: 1,
-            complete: this.complete,
```

Comment:
Hrm, if you move onto the next question and then refresh the page, it returns to the errored question.
I think it might be best to keep this as `this.complete`, and just set `correct` to 0.
Finally, at the end of the `handleItemError` method, call `this.saveAttemptLogMasterLog();`.